docs/codingconventions.adoc

```diff
@@ -0,0 +1,16 @@
+= Coding conventions
+
+This is a list of coding conventions you should observe when contributting code for this project.
+
+Addressed subjects are the following:
+
+- adopting the right case type
+- good names for files and variables
+
+It is not an exhaustive list and it will be updated as we go. In the mean time, your PR reviewer may suggest conventions that are not listed here. Please update.
+
+== Names
+
+- Use snake_case when naming variables and resources
+- If your variable is controlling a behaviour, use imperative style to name it e.g. create_internet_gateway
+- Use simple file names where possible; avoid '-', '' for terraform files. '' is ok for bash/python.
```

Comment:
What is a simple file name? Why avoid `-`?  Also, I think your second and third characters (a space perhaps) have been eaten by the markdown processor. You'll have to wrap `'-'` and `' '` in back ticks to get the space to render. If it is a space, it's not recommended for shell scripts or Python either.

Reply:
We may just suggest to stick with snake_case for Terraform file names.